### PR TITLE
added working dir /app

### PR DESCRIPTION
**Pull request by ivanminang**
Working Dir added

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,2 +1,3 @@
 FROM httpd:2.4
 COPY ./public-html/ /usr/local/apache2/htdocs/
+WORKDIR /app
```
Comment (devopseasylearning) [inline Dockerfile]: PLease remove empty  line

**Review by ivanminang — COMMENTED** [review]
@devopseasylearning i need you to review please

**Review by NtBahati — COMMENTED** [review]
I got a few questions. Can you please clarify your commit message. It is confusing.

**Comment by ivanminang** [comment]
> I got a few questions. Can you please clarify your commit message. It is confusing.

I added a working directory named /app to the Dockerfile

**Comment by ivanminang** [comment]
@s7testar @S7belliss @NtBahati @sandargo can you approve please

File: Dockerfile [hunk 2]
```diff
@@ -1,18 +1,13 @@
 FROM httpd:2.4
 COPY ./public-html/ /usr/local/apache2/htdocs/
-
 WORKDIR /app
-
 # Set metadata for the container
 LABEL phone="800-989-5000" 
 LABEL address="14225 DevOps Success Rd, Houston, 
 LABEL email="info@devops.com"
-
```
Comment (devopseasylearning) [inline Dockerfile:10]: This is not how to set env in dockerfile please visit https://docs.docker.com/engine/reference/builder/

**Comment by s7eric** [comment]
good job

**Review by sdjakou — APPROVED** [review]

**Review by s7eric — APPROVED** [review]

**Review by sadioci2 — APPROVED** [review]
approved by sadioci2

**Review by THIMONE-NOUPA — APPROVED** [review]

**Review by EDEMOS80 — COMMENTED** [review]
good togo

**Review by s7eric — APPROVED** [review]

**Review by sandargo — APPROVED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -1,18 +1,13 @@
...
 APP=DEVINE
```
Comment (devopseasylearning) [inline Dockerfile:8]: https://docs.docker.com/engine/reference/builder/ 
that is not how you set env in dockerfile check lin above

**Comment by devopseasylearning** [comment]
@s7eric @sandargo please withdraw your approval this will break the container to build

File: Dockerfile (hunk 3 above)
Comment (ivanminang) [reply Dockerfile:8]: @devopseasylearning  can you resolve the conversation please

File: Dockerfile (hunk 1 above)
Comment (ivanminang) [reply Dockerfile]: @devopseasylearning  can you also close this one please

File: Dockerfile (hunk 3 above)
Comment (devopseasylearning) [reply Dockerfile:8]: did you solved my concern already ?

**Review by s7eric — APPROVED** [review]

File: Dockerfile (hunk 3 above)
Comment (devopseasylearning) [reply Dockerfile:8]: image

Comment (devopseasylearning) [reply Dockerfile:8]: image

Comment (devopseasylearning) [reply Dockerfile:8]: 513E120C-1835-4E47-8587-E3D0F36A6E1D_4_5005_c

Comment (devopseasylearning) [reply Dockerfile:8]: https://docs.docker.com/engine/reference/builder/

File: Dockerfile [hunk 4]
```diff
@@ -1,18 +1,12 @@
...
 SECTION=V1 
 HOST=devopseasylearning.com
 PORT=5278 
 ZONE=PROD
```
Comment (devopseasylearning) [inline Dockerfile:12]: there is an issue here

Comment (devopseasylearning) [reply Dockerfile:12]: https://docs.docker.com/engine/reference/builder/#env

Comment (devopseasylearning) [reply Dockerfile:12]: 513E120C-1835-4E47-8587-E3D0F36A6E1D_4_5005_c

Comment (devopseasylearning) [reply Dockerfile:12]: please review the link above

Comment (devopseasylearning) [reply Dockerfile:12]: any update ?
do you need assistance 
we need this ready for Monday deployment 
thank you for all the hard and wonderful work

File: Dockerfile [hunk 5]
```diff
@@ -1,18 +1,12 @@
 FROM httpd:2.4
 COPY ./public-html/ /usr/local/apache2/htdocs/
-
 WORKDIR /app
-
 # Set metadata for the container
 LABEL phone="800-989-5000" 
 LABEL address="14225 DevOps Success Rd, Houston,
```
Comment (devopseasylearning) [inline Dockerfile:6]: there  is a quote `"` missing here

Comment (devopseasylearning) [reply Dockerfile:6]: also why are you ending the sentence with `,` ?

**Review by sadioci2 — APPROVED** [review]

**Pull request merged**